config/config.json

```diff
@@ -0,0 +1,13 @@
+{
+    "database" : {
+        "port": "27017",
+        "uri": "localhost",
+        "collection": "PokeData"
+    },
+    "twitter" : {
+        "consumer_key": "46yqEVwOOkf7dtN5Zfqxw4Nzu",
```

Comment:
@samitsv hey, maybe you forgot to push again after revoking the credentials because they still seem to be there.